packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -72,6 +75,13 @@ export class PharosImageCard extends LitElement {
   @property({ type: String, reflect: true, attribute: 'action-menu' })
   public actionMenu?: string;
 
+  /**
+   * The heading level to use for the card title when using the title property.
+   * @attr heading-level
```

Comment:
```suggestion
   * @attr heading-level
   * @type {HeadingLevel | undefined}
```
So our wrapper knows to make this optional

Reply:
Looks like that's failed the tests—taking a look

Reply:
Think you also have to export the type

`export type { HeadingLevel };`

Reply:
I think it's unhappy that the component doesn't `export type HeadingLevel` (it's defined in `pharos-heading`). Can/should we `export` it from this module as well? 🤔

Reply:
Yup!